Fluid/Ast/ForStatement.cs

```diff
@@ -25,7 +24,7 @@ public sealed class ForStatement : TagStatement
             Reversed = reversed;
             Else = elseStatement;
 
-            _isContinueOffset = Offset is MemberExpression l && l.Segments.Count == 1 && ((IdentifierSegment)l.Segments[0]).Identifier == "continue";
+            OffSetIsContinue = Offset is MemberExpression l && l.Segments.Count == 1 && ((IdentifierSegment)l.Segments[0]).Identifier == "continue";
```

Comment:
Can we used `static` field for "Continue" identifier

Reply:
If it was used somewhere else it could be a const, but it just local so it doesn't matter.